packages/zowe-explorer-api/src/profiles/ProfilesCache.ts

```diff
@@ -194,6 +194,7 @@ export class ProfilesCache {
     public async refresh(apiRegister?: ZoweExplorerApi.IApiRegisterClient): Promise<void> {
         this.allProfiles = [];
         this.allTypes = [];
+        this.profilesByType.clear();
```

Comment:
Thanks for adding this 🙂 Should we also clear the `defaultProfileByType` map?

Reply:
No problem - and yes, thanks for catching that, addressed in `ce52b02`